doc/source/reference/distutils.rst

```diff
@@ -117,6 +120,11 @@ install the C library, you just use the method `add_installed_library` instead o
 `add_library`, which takes the same arguments except for an additional
 ``install_dir`` argument::
 
+.. for doctests
+
```

Comment:
I think this would read easier without the blank lines.

Reply:
sphinx complains about the version without a blank line, see failed circle ci

Reply:
Interesting. I based the suggestion on other uses of `.. for doctests` in the documentation and it seems they may be in error.

Reply:
Actually, looks like the indentation of one line is off.

Reply:
How were you able to resolve the warning, @pdebuyl?

Reply:
By indenting the comment marker by two extra spaces.